<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficRecordImportWindow</class>
 <widget class="QDialog" name="TrafficRecordImportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="importGuideStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="beforeImportPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="beforeImportHeaderLabel">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Befor you Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="beforeImportSubheaderLabel">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>We are specific about the format</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="makeSureLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Please make sure:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="currectDataExtentionCheck">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>The file ends with .xlsx (Microsoft Excel)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="currectFormatCheck">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>The spreadsheet has the following format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTableWidget" name="exampleFormatTableWidget">
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>Record Date</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pole Code</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Speed</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Registration No</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>25/04/2021  09:44:48</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>K31+950</string>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>43.12</string>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string>BYZ858</string>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>25/04/2021  09:46:41</string>
               </property>
              </item>
              <item row="1" column="1">
               <property name="text">
                <string>K31+950</string>
               </property>
              </item>
              <item row="1" column="2">
               <property name="text">
                <string>42.29</string>
               </property>
              </item>
              <item row="1" column="3">
               <property name="text">
                <string>A289LZ</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>128</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="cancelButton_beforeImportPage">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton_beforeImportPage">
           <property name="text">
            <string>Next &gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="filePathPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="filePathLabel">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Where is your file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="filePathSubheaderLabel">
         <property name="text">
          <string>Select the file you wish to import</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTreeView" name="filePathTreeView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="dataCountLabel">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>How many records does it contain?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="dataCountEdit">
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="previousButton_filePagePage">
           <property name="text">
            <string>&lt;&lt; Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton_filePathPage">
           <property name="text">
            <string>Next &gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="donePage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Almost done</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Let's review the data imported</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTableView" name="importedRecordTableView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="cancelButton_donePage">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton_donePage">
           <property name="text">
            <string>Confirm</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
